pkg/detectors/microsoftteamswebhook/microsoftteamswebhook.go

```diff
@@ -27,7 +27,7 @@ var (
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"microsoft"}
+	return []string{"microsoft"} // should we instead look for webhook.office.com, or something required to be in the regex above?
```

Comment:
This is unrelated to the overall goal, so I'm happy to pull it into another PR. But I think if the key must have `webhook.office.com` in the key pattern, that's probably fine to use as the keyword as well.

Reply:
Yes, that should replace the existing keyword. It is ideal to use something that must be in the pattern if possible.